packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1865,6 +1865,12 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Once [markNeedsPaint] has been called on a render object,
   /// [debugNeedsPaint] returns true for that render object until just after
   /// the pipeline owner has called [paint] on the render object.
+  ///
+  /// See also:
+  ///
+  ///  * [RepaintBoundary], to contain render objects marked with
+  ///    [markNeedsPaint] to only render subtrees that visually changed for best
+  ///    performance.
```

Comment:
How about: `to scope a subtree of render objects to their own layer, thus limiting the number of nodes that [markNeedsPaint] must mark dirty.` ?

Reply:
Ok